tests/functional/lib/helpers.js

```diff
@@ -191,6 +191,22 @@ define([
     }, [ selector ], timeout);
   }
 
+  function visibleByQSAHeight(selector, timeout, minHeight) {
```

Comment:
maybe we should do `selector, options` to avoid passing `undefined` below. 

This way if we want to tweak this later it will be easier